packages/calypso-data-stores/tsconfig.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
Copied from `@automattic/components`

Reply:
It would likely be better to look at
- https://github.com/Automattic/wp-calypso/blob/72fc8477fc10bcbb4848ad36cb29df49c945fdf3/packages/composite-checkout-wpcom/package.json#L9-L14
- https://github.com/Automattic/wp-calypso/blob/72fc8477fc10bcbb4848ad36cb29df49c945fdf3/packages/composite-checkout-wpcom/tsconfig.json
- https://github.com/Automattic/wp-calypso/blob/72fc8477fc10bcbb4848ad36cb29df49c945fdf3/packages/composite-checkout-wpcom/tsconfig-cjs.json